docs/docs/getting_started.md

```diff
@@ -2,43 +2,30 @@
 title: Quickstart
 ---
 
-The easiest way to start developing on Aztec is simply to click on one of these buttons:
+The easiest way to start developing on Aztec locally is through `npx aztec-app`. This is a convenient way of installing the development environment (A.K.A. Sandbox) and starting new projects from a boilerplate.
 
-[![One-Click React Starter](/img/codespaces_badges/react_cta_badge.svg)](https://codespaces.new/AztecProtocol/aztec-packages?devcontainer_path=.devcontainer%2Freact%2Fdevcontainer.json) [![One-Click HTML/TS Starter](/img/codespaces_badges/vanilla_cta_badge.svg)](https://codespaces.new/AztecProtocol/aztec-packages?devcontainer_path=.devcontainer%2Fvanilla%2Fdevcontainer.json) [![One-Click Token Starter](/img/codespaces_badges/token_cta_badge.svg)](https://codespaces.new/AztecProtocol/aztec-packages?devcontainer_path=.devcontainer%2Ftoken%2Fdevcontainer.json)
+If you'd like to develop remotely (for example, if you're on Windows or have trouble working with Docker), follow the [codespaces guide](./getting_started/codespaces.md).
 
-That's it!
-
-This creates a codespace with a prebuilt image containing one of the "Aztec Boxes" and a development network (sandbox). 
-- You can develop directly on the codespace, push it to a repo, make yourself at home.
-- You can also just use the sandbox that comes with it. The URL will be logged, you just need to use it as your `PXE_URL`.
-
-## Develop Locally
-
-The above method uses Aztec boxes to install the sandbox and clone the repo. You can use it too to get started on your own machine and use your own IDE.
-
-You can also [install the sandbox manually](/reference/sandbox_reference).
-
-### Prerequisites
+## Prerequisites
 
 - Node.js >= v18 (recommend installing with [nvm](https://github.com/nvm-sh/nvm))
 - Docker (visit [this page of the Docker docs](https://docs.docker.com/get-docker/) on how to install it)
 
 ### Run the `npx` script
 
-With the node installation, you now should have `npm` and be able to run `npx` scripts. You can do that by running:
+Thanks to Node, you can run the recommended `npx script`:
 
 ```bash
-npx create-aztec-app
+npx aztec-app
 ```
 
-And follow the instructions. If all goes well, you should now have a development environment running locally on your machine.
-
-You can run `npx create-aztec-app sandbox -h` to start, stop, update and output logs from the sandbox. 
+This script gives you some options to bootstrap a new project, start/stop the sandbox, or see the logs. Run `npx aztec-app -h` for a list of options.
```

Comment:
Is there a list of all of the commands available through `aztec-app` and what they do anywhere? If not, this would be good to add.

Reply:
There isn’t, I thought about adding but that would be one more thing to maintain. Didn’t seem worth it at this point, given you can just call it with the -h flag